js/indicia.datacomponents/jquery.idc.esDownload.js

```diff
@@ -300,6 +300,25 @@
       $.extend(currentRequestData, columnSettings);
       // Reset.
       rowsToDownload = null;
+      // If there is an associated download template select control,
+      // set the download template option from its value.
+      if ($('#' + el.id + '-template').val()) {
+        currentRequestData['columnsTemplate'] = $('#' + el.id + '-template').val();
+      }
+      // Set columnsSurveyId to survey ID if appropriate.
+      if (typeof(el.settings.columnsSurveyId) != "undefined") {
```

Comment:
Coding standards, either:
```js
if (typeof(el.settings.columnsSurveyId) !== 'undefined') {
```
or I think this will do the trick:
```js
if (el.settings.columnsSurveyId)
```

Reply:
I've used `!== 'undefined'` because I need to treat differently if columnSurveyId is specified but given not value.